docs/ucr/mobile_ucr_v2_migration_guide.rst

```diff
@@ -0,0 +1,83 @@
+Migrating to Mobile UCR Restore V2
+==================================
+
+Evolution of Mobile UCR Restore Versions
+----------------------------------------
+Historically, there have been three versions for how Mobile UCRs were sent to mobile devices:
+
+- **Version 1.0**: All reports are bundled together into a single fixture during the restore process.
+- **Version 1.5**: Both V1 and V2 are supported together. This is used to migrate from V1 without causing any breaking changes.
+- **Version 2.0**: Enhanced performance where each report has a fixture of its own.
+
+Deprecation of Mobile UCR V1 and V1.5
+-------------------------------------
+Both V1 and V1.5 have been deprecated to address the performance challenges encountered with these versions.
+
+Benefits of Upgrading to Mobile UCR V2
+--------------------------------------
+The introduction of Mobile UCR V2 offers significant performance and efficiency improvements around how UCR data is restored and synced.
```

Comment:
nit: restored -> sent over to mobile app.

Reply:
All mentions of "restore" have been replaced with "mobile sync" in [fe7b75a](https://github.com/dimagi/commcare-hq/pull/35276/commits/fe7b75a3b6897a234a1eab70bbbeb95ec325d8b1).